__docs__/wonder-blocks-form/text-field.argtypes.ts

```diff
@@ -187,6 +187,19 @@ export default {
         },
     },
 
+    instantValidation: {
```

Comment:
Wasn't able to get the autodocs working with TextField, we have WB-1765 to remove prop descriptions from *.argtypes.ts files!

Reply:
hmm yeah that's probably because `TextField` uses union types, which is a react-docgen limitation as it cannot infer those kind of complex prop types.

For reference: https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-form/src/components/text-field.tsx#L149